### PR TITLE
(BKR-210) allow empty options as a default instead of required parameter

### PR DESCRIPTION
**Pull request by cyberious**

### CODE REVIEW

File: lib/beaker/dsl/install_utils/module_utils.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def copy_module_to(one_or_more_hosts, opts = {})
                 end
               end
             when 'rsync'
-              rsync_to host, source, File.join(target_module_dir, module_name), {:ignore => ignore_list}
+              rsync_to host, source_path, File.join(target_module_dir, module_name), {:ignore => ignore_list}
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils/module_utils.rb]: Looks like you've got a ride-along here for BKR-189, could you indicate that in your commit message, please?

Comment (cyberious) [inline lib/beaker/dsl/install_utils/module_utils.rb]: Updated commit message for BKR-210 to include (BKR-189) in second line of commit message

**Comment by kevpl** [comment]
@cyberious this is failing spec tests.  Let us know if you're not seeing the same thing locally.

**Comment by anodelman** [comment]
Spec failures:

```
Failures:

  1) ClassMixedWithDSLInstallUtils#install_puppet_module_via_pmt_on installs module via puppet module tool
     Failure/Error: subject.install_puppet_module_via_pmt_on( master, {:module_name => 'test'} )
       #<ClassMixedWithDSLInstallUtils:0x00000006538620> received :puppet with unexpected arguments
         expected: ("module install test ", {})
              got: ("module install - ", {})
       Diff:
       @@ -1,2 +1,2 @@
       -["module install test ", {}]
       +["module install - ", {}]
       
     # ./lib/beaker/dsl/install_utils/module_utils.rb:79:in `block in install_puppet_module_via_pmt_on'
     # ./lib/beaker/shared/host_manager.rb:95:in `run_block_on'
     # ./lib/beaker/dsl/patterns.rb:32:in `block_on'
     # ./lib/beaker/dsl/install_utils/module_utils.rb:65:in `install_puppet_module_via_pmt_on'
     # ./spec/beaker/dsl/install_utils/module_utils_spec.rb:86:in `block (3 levels) in <top (required)>'

  2) ClassMixedWithDSLInstallUtils#install_puppet_module_via_pmt_on takes the trace option and passes it down correctly
     Failure/Error: subject.install_puppet_module_via_pmt_on( master, {:module_name => 'test'} )
       #<ClassMixedWithDSLInstallUtils:0x00000006611f88> received :puppet with unexpected arguments
         expected: ("module install test ", {:trace=>nil})
              got: ("module install - ", {:trace=>nil})
       Diff:
       @@ -1,2 +1,2 @@
       -["module install test ", {:trace=>nil}]
       +["module install - ", {:trace=>nil}]
       
     # ./lib/beaker/dsl/install_utils/module_utils.rb:79:in `block in install_puppet_module_via_pmt_on'
     # ./lib/beaker/shared/host_manager.rb:95:in `run_block_on'
     # ./lib/beaker/dsl/patterns.rb:32:in `block_on'
     # ./lib/beaker/dsl/install_utils/module_utils.rb:65:in `install_puppet_module_via_pmt_on'
     # ./spec/beaker/dsl/install_utils/module_utils_spec.rb:97:in `block (3 levels) in <top (required)>'

Finished in 14.68 seconds (files took 1.86 seconds to load)
867 examples, 2 failures
```

Also, needs rebase.

**Comment by anodelman** [comment]
Last review comment still current.

**Comment by anodelman** [comment]
Will close out next week due to inactivity.

**Comment by anodelman** [comment]
Closing, inactive.

**Pull request closed without merging**